### PR TITLE
Feature/rfi improvements

### PR DESCRIPTION
**Pull request by afeena**
Store scripts as a files, return to the SNARE only stdout

### CODE REVIEW

File: rfi_emulator.py [hunk 1]
```diff
@@ -1,54 +1,70 @@
...
+                filename = hashlib.md5(data.encode('utf-8')).hexdigest()
+                with open(self.script_dir + filename, 'w') as rfile:
+                    rfile.write(data)
+            finally:
+                yield from resp.release()
+                return filename
 
     @asyncio.coroutine
-    def get_rfi_result(self, script):
+    def get_rfi_result(self, file):
```
Comment (glaslos) [inline rfi_emulator.py]: `file` shadows the build in `file` method

File: rfi_emulator.py [hunk 2]
```diff
@@ -1,54 +1,70 @@
...
-            print('bad phpox server link : %s' % e)
-        else:
-            yield from resp.release()
-        finally:
-            session.close()
-            return rfi_result
+        with aiohttp.ClientSession(loop=loop) as session:
+            resp = yield from session.post('http://127.0.0.1:8088/', data=script)
+            try:
+                rfi_result = yield from resp.text()
```
Comment (glaslos) [inline rfi_emulator.py]: There should be a `resp.json()`

File: server.py [hunk 3]
```diff
@@ -37,7 +38,7 @@ class HttpRequestHandler(aiohttp.server.ServerHttpProtocol):
 
     def __init__(self, *args, **kwargs):
         super(HttpRequestHandler, self).__init__()
-        self.rfi_emulator = RfiEmulator()
+        self.rfi_emulator = RfiEmulator(os.getcwd())
```
Comment (glaslos) [inline server.py]: `/opt/tanner/`

**Comment by glaslos** [comment]
Can you rebase this branch?

**Comment by afeena** [comment]
rebased

**Pull request merged**